resources/pages/login_pages.robot

```diff
@@ -5,7 +5,22 @@ Resource    ${EXECDIR}/resources/locators/login_locators.robot
 
 *** Keywords ***
 
+#########################################################################
+
+Clicar em um item específico do Menu da página de Login
+    [Arguments]     ${text_search}
+    ${element} =    String Replace  ${MENU_OPTION}  ${text_search}
+
+    Wait Until Element Is Visible   ${element}
```

Comment:
A keyword Click If Element Is Visible já está implementando a validação de visibilidade do elemento. Este step é desnecessário.